core-services/prow/02_config/openshift-pipelines/console-plugin/_prowconfig.yaml

```diff
@@ -0,0 +1,14 @@
+tide:
+  queries:
+  - labels:
+    - approved
+    - lgtm
```

Comment:
these label should be required for both features and bugs? Asking since we have a different seet of labels which need to be applied for bug or story.

Reply:
I think so 🤔 At least this is what we do upstream as well (in tektoncd org), `approve` and `lgtm` are required, no matter the type of PR.